src/runtime/server/server.ts

```diff
@@ -183,37 +183,28 @@ const wrapHandlerWithErrorHandling = (handler: NexusRequestHandler): NexusReques
   }
 }
 
-type AnonymousRequest = Record<string, any>
-
-type AnonymousContext = Record<string, any>
-
-export type ContextCreator<
-  Req extends AnonymousRequest = AnonymousRequest,
-  Context extends AnonymousContext = AnonymousContext
-> = (req: Req) => MaybePromise<Context>
-
 /**
  * Combine all the context contributions defined in the app and in plugins.
  */
 function createContextCreator(
   contextContributors: ContextContributor[],
   plugins: Plugin.RuntimeContributions[]
-): ContextCreator {
-  const createContext: ContextCreator = async (req) => {
+): ContextContributor {
+  const createContext: ContextContributor = async (params) => {
     let context: Record<string, any> = {}
 
     // Integrate context from plugins
     for (const plugin of plugins) {
       if (!plugin.context) continue
-      const contextContribution = await plugin.context.create(req)
+      const contextContribution = await plugin.context.create(params.req)
```

Comment:
Put a todo to update plugin lens as well? iiuc you're maintaining its contract for now.

Reply:
Correct! I figured out it wasn't super important yet. I can also break the plugin interface now if you think it's better

Reply:
I think todo is enough for now. Don't want to forget to consider this in the future, but don't see much benefit right now either.